gen/nvml/nvml.h

```diff
@@ -627,9 +627,9 @@ typedef struct
     struct
     {
         nvmlThermalController_t controller;
-        unsigned int defaultMinTemp;
-        unsigned int defaultMaxTemp;
-        unsigned int currentTemp;
+        int defaultMinTemp;
```

Comment:
This is internal only from our perspective, correct?

Reply:
What do you mean by internal only? The file here in the `gen` directory should be a direct copy from what's upstream.

Reply:
I mean that the change of type doesn't require other changes in our wrapper.

Reply:
It shouldn't since they are struct fields set by the end user, not us.